### DISCUSSION TITLE
How to see security-context of container..?

### DISCUSSION THREAD

**Discussion by moon2263** [containerd/nerdctl#2096] [Q&A]
Hi, I'm looking for some way to see the container's security-context like privileged, network/pid/ipc namespace mode.
When I use docker, I can see that configurations in `docker inspect <container>`. But in containerd environment, If I use the command `nerdctl inspect --mode=native` there was no any information about container's security-context. 
So,, Is there any way to see that information to using nerdctl?

### What I expected is
<img width="382" alt="image" src="https://user-images.githubusercontent.com/35019532/224754008-f4be81d8-ea6c-4328-aea6-33b569efcd74.png">
(this is crictl's result)

p.s. I'm not goot at english. sorry :(

**Comment by fahedouch** [comment]
Hi @moon2263 , I believe that these infos are present but not in the same format as the above output

I can see the added caps to the container spec `nerdctl inspect --mode=native` :

```
                "capabilities": {
                    "bounding": [
                        "CAP_CHOWN",
                        "CAP_DAC_OVERRIDE",
                        "CAP_DAC_READ_SEARCH",
                        "CAP_FOWNER",
                        "CAP_FSETID",
                        "CAP_KILL",
                        "CAP_SETGID",
                        "CAP_SETUID",
                        "CAP_SETPCAP",
                        "CAP_LINUX_IMMUTABLE",
```

**Reply by moon2263** [reply]
Oh, I'm sorry I confused you.. 
yes right. I can see that caps info on my containers. But what I wanna see is privileged or network/pid/ipc namespace info,,
Thank you for your reply!

**Reply by fahedouch** [reply]
Hi @moon2263 , Sorry for the delayed response. Would you please create an issue ticket ?
